docs/developerguide.rst

```diff
@@ -127,38 +148,27 @@ To setup the GitLab integration locally you need to follow these steps:
 
   Select ``api`` scope and *Save application*.
 
-2. Configure GitLab-related REANA-Cluster environment variables. You can do this directly on the
-   `reana-cluster.yaml <https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/configurations/reana-cluster-minikube-dev.yaml>`_:
+2. Configure GitLab-related environment variables. You can do this directly on the
+   `reanahub/reana`'s `helm/reana-dev/values.yaml <https://github.com/reanahub/reana/blob/master/helm/reana/values.yaml>`_:
 
 .. code-block:: diff
 
-      cluster:
-      type: "kubernetes"
       + reana_url: <hostname> # e.g. "reana-johndoe.cern.ch"
+      debug:
+        enabled: false
+      ui:
+        enabled: false
       ...
-      reana-workflow-controller:
-      type: "docker"
-      image: "reanahub/reana-workflow-controller:latest"
-      mountpoints:
-        - type: hostPath
-        name: reana-workflow-controller-code
-        path: "/code/reana-workflow-controller:/code"
-      environment:
-      ...
-      + - REANA_GITLAB_HOST: "gitlab-test.cern.ch"
-      ...
-      reana-server:
-      type: "docker"
-      image: "reanahub/reana-server:latest"
-      mountpoints:
-        - type: hostPath
-        name: reana-server-code
-        path: "/code/reana-server:/code"
-      environment:
+      secrets:
+        ...
+        gitlab:
+      -    REANA_GITLAB_OAUTH_APP_ID: reana
+      +    REANA_GITLAB_OAUTH_APP_ID: reana
+      -    # REANA_GITLAB_OAUTH_APP_SECRET: <gitlab_oauth_app_secret>
+      +    REANA_GITLAB_OAUTH_APP_SECRET: "<real-GitLab-Secret>"
```

Comment:
`<real_gitlab_oauth_app_secret>`?

Reply:
:+1: One thing I always like is having a certain prefix for certain things to facilitate finding similar stuff via `git grep real_`.  

For example, we use `FIXME` in the code, so that `pylint` and friends will report these lines, but also so that developers can do anytime `git grep FIXME` to find all the places where further work is being tagged.  Having one standardised string is preferable to having `FIX ME`, `TODO`, and a plethora of other such similar tags.  

In the similar manner, we can invent a common prefix such as `real_...` and use it everywhere, as then we could say in the docs that "admins can look for this string to customise stuff".  

BTW note that elsewhere we used `CHANGEME` prefix for these things, it so could be also `CHANGEME_real_gitlab_oauth_app_secret`... but we also used `CHANGE_ME`, which is not cool, because `git grep` stops working. 

Let's harmonise the "customize this" prefix on some value and use it the same everywhere...